### PR TITLE
Move DatabaseFeatures to features module

### LINKED ISSUE #15: Move DatabaseFeatures to features module
Most Django backends have the Database features in a separate module than base.
https://github.com/django/django/tree/master/django/db/backends

### PR DESCRIPTION
**Pull request by naveen-ram**
Fixes #15

### CODE REVIEW

**Comment by kadler** [comment]
Can we rename the package first (#12) first before we start shuffling stuff around?

File: django_ibmi/features.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from django.db.backends.base.features import BaseDatabaseFeatures
```
Comment (kadler) [inline django_ibmi/features.py:1]: We're no longer trying django.db.backends. Is that a pre Django 2 package?

File: django_ibmi/base.py [hunk 2]
```diff
@@ -24,12 +24,6 @@
 from django.core.exceptions import ImproperlyConfigured
 
 # Importing class from base module of django.db.backends
```
Comment (kadler) [inline django_ibmi/base.py]:
```suggestion
```

Dead comment

File: django_ibmi/features.py (hunk 1 above)
Comment (naveen-ram) [reply django_ibmi/features.py:1]: copying the way it is done by all the backends here https://github.com/django/django/tree/stable/3.0.x/django/db/backends

Comment (kadler) [reply django_ibmi/features.py:1]: Ok, makes sense

**Review by kadler — APPROVED** [review]

**Pull request merged**